### DISCUSSION TITLE
Using chatgpt web instead of API.

### DISCUSSION THREAD

**Discussion by mkasajim** [AntonOsika/gpt-engineer#938] [Ideas]
Only some people have API access, but everyone has ChatGPT access for free, even a lot of people have ChatGPT premium access but do not have API access. So, instead of using API, can you please make an option to use ChatGPT web? Some browser extensions do that already (for example - HarpaAI), so is it possible?

**Comment by ErikBjare** [comment]
This would be a lot of work and difficult to implement. The free version of ChatGPT (3.5) won't work very well anyway.

It is fairly quick & easy to get API access these days, so I suggest you go that route instead.

**Comment by captivus** [comment]
Closing, and echoing @ErikBjare's notes. Thanks for the idea, but please sign-up for API access and enjoy using GPTE!
